[WIP] Stream page

alex opened the pull request:
Fixes #2.

alex committed:
Initial skeleton HTML for stream page.

Fixes #2.

alex committed:
a TODO

strcrzy commented:
👍

liyanchang committed:
Add basic css, fonts, logos

liyanchang committed:
Adding Makefile (#8)

* Adding gitignore

* Adding makefile for dependencies, local, deploy

* Add README

nothingisfunny committed:
First attempt at styling the page

liyanchang committed:
basic grid

liyanchang committed:
format form

liyanchang committed:
Add padding on sidebar; Center logo

liyanchang committed:
Modifying skeleton to respond at different width

Shouldn't edit vendor files but didn't really see an easy way to do so
and didn't see any major downside.

liyanchang committed:
Add title

liyanchang committed:
Add link styles; color palette

liyanchang committed:
background colors

liyanchang committed:
Passing CSS linting

liyanchang committed:
Pass accessibility lint

liyanchang committed:
Default gulp and make actions

liyanchang committed:
Configure CircleCI

nothingisfunny committed:
remove display:table

liyanchang committed:
Full height sidebar

nothingisfunny committed:
add media query to make more responsive

liyanchang committed:
Pass html5 linter but did not add to gulp

liyanchang committed:
Make mobile devices responsive

liyanchang committed:
Mobile device issues

liyanchang committed:
Add favicon

liyanchang committed:
Merge pull request #10 from nothingisfunny/style

First attempt at styling the page

liyanchang committed:
Revert "First attempt at styling the page"

liyanchang committed:
Merge pull request #12 from kpenning/revert-10-style

Revert "First attempt at styling the page"

liyanchang committed:
Merge pull request #9 from kpenning/ldchang-style

Adding basic styling

liyanchang committed:
Merge branch 'ldchang-lint' into stream-page

samantharitter committed:
Change style on page for new layout

liyanchang committed:
Linters (#17)

* Remove Gulp

* CSS Lint with Grunt

* Accessibility linter

* html linter

liyanchang committed:
Last mile wiring for linters

liyanchang committed:
linter

liyanchang committed:
First pass

liyanchang committed:
update logo

liyanchang committed:
Merge pull request #18 from kpenning/stream-page-sam

Stream page sam

liyanchang committed:
Seeing if I'm happier with bootstrap

liyanchang committed:
Happier with a lightweight bootstrap

liyanchang committed:
sticky footer for large screens

liyanchang committed:
button fix

liyanchang committed:
Merge pull request #19 from kpenning/ldchang-bootstrap

Modified page

liyanchang commented:
Try it yourself: https://s3.amazonaws.com/ldchang-peoplepoweredstream/index.html

Desktop:
![screenshot 2017-03-08 00 40 10](https://cloud.githubusercontent.com/assets/328073/23691742/50ae7070-0398-11e7-8ce4-ea60c3a2a989.png)
Phone:
![screenshot 2017-03-08 00 39 57](https://cloud.githubusercontent.com/assets/328073/23691743/50b4310e-0398-11e7-8dfa-3a348aa26142.png)
Tablet:
![screenshot 2017-03-08 00 40 05](https://cloud.githubusercontent.com/assets/328073/23691744/50b43924-0398-11e7-86fa-6263f57a2c54.png)

liyanchang committed:
adding jquery as samantha intended

liyanchang committed:
ie media query responsive fix

liyanchang committed:
Merge pull request #21 from kpenning/ldchang-js

Adding JS vendor files

liyanchang committed:
Add JSHint

liyanchang committed:
Merge pull request #22 from kpenning/ldchang-jshint

Add JSHint

liyanchang committed:
Using ie8 compatible version of jQuery

liyanchang committed:
Merge pull request #23 from kpenning/ldchang-jquery1

Using ie8 compatible version of jQuery

alex committed:
Wired up initial action kit submission. (#24)

This hooks it up using the code Kenneth provided, it definitely submits to ActionKit.

It's not Ajax though, so navigation takes you away from the stream page. Not clear me to how to do otherwise.

liyanchang committed:
Add video click to play and background image

liyanchang committed:
Pass linters

liyanchang committed:
video should size down

liyanchang committed:
simplify js

liyanchang committed:
correct privacy policy page

alex committed:
Use HTML5 required attribute on required inputs (#30)

liyanchang committed:
Correct ActionKit JS error

liyanchang committed:
Merge pull request #29 from kpenning/ldchang-video-click

Add video click to play and background image

liyanchang committed:
Use ActionKit validation

While I would prefer HTML5 elements, will choose to just use action kit
validation.

liyanchang committed:
Override the form submission with ajax

alex committed:
Fixed #36 changed the button text

alex committed:
Fixed #37 -- added extra opt-in text

mollieru committed:
Update style.css

CSS fixes to:
• h4 font size
• body font size
• btn letter spacing
• legend font weight

stuartsan committed:
Webfonts

stuartsan committed:
Rename

stuartsan committed:
Fix linting error by removing the rule ha ha

strcrzy committed:
Merge pull request #31 from kpenning/ldchang-ajax

Override the form submission with ajax

strcrzy committed:
address PR #31 feedback

- dynamically restore sign up button text to original value
- break callback handler into two functions

strcrzy committed:
Merge pull request #39 from stuartsan/stuart/fonts

Use webfonts

strcrzy committed:
address PR feedback

strcrzy committed:
Merge pull request #40 from kpenning/strcrzy-ajax-revised

address PR #31 feedback

stuartsan committed:
Maintain aspect ratio

Commit:
Show past events temporarily

stuartsan committed:
Reduce unnecessary diff

kpenning committed:
Merge pull request #41 from stuartsan/hero-height-aspect-ratio

Maintain aspect ratio

Commit:
Style changes and alert bar for before the event

stuartsan committed:
oops where r u autoprefixer

liyanchang committed:
Merge pull request #42 from stuartsan/ie9-oops

IE9 fix

liyanchang committed:
Commit 18ecb48a added the right url

liyanchang committed:
css typo

liyanchang committed:
remove tabs from css

liyanchang committed:
alphabetical order for css

liyanchang committed:
last css lints

liyanchang committed:
remove tabs from html

liyanchang committed:
css over inline style

liyanchang committed:
js semicolons

liyanchang committed:
Merge pull request #43 from kpenning/ldchang-clean2

Lint cleanup
